Soomsil-USaint/Application/Feature/Home/View/HomeView.swift

```diff
@@ -1,335 +1,187 @@
-                Text("/ \(String(format: "%.2f", 4.50))").font(YDSFont.subtitle3)
-                    .foregroundColor(Color(red: 0.56, green: 0.58, blue: 0.6))
-                    .padding(.leading, -4)
-            }
-            .frame(height: 23)
-            .padding(.vertical, Dimension.DetailPadding.vertical)
-            .padding(.horizontal, Dimension.DetailPadding.horizontal)
-
-            Divider()
-                .padding(.horizontal, 13)
-
-            HStack {
-                Text("취득학점").font(YDSFont.body1)
-                Spacer()
-                Text(String(format: "%.1f", credit)).font(YDSFont.subtitle2)
-                    .foregroundColor(Color(red: 0.51, green: 0.43, blue: 0.93))
-                Text("/ \(String(Int(graduateCredit)))").font(YDSFont.subtitle3)
-                    .foregroundColor(Color(red: 0.56, green: 0.58, blue: 0.6))
-                    .padding(.leading, -4)
-            }
-            .frame(height: 23)
-            .padding(.vertical, Dimension.DetailPadding.vertical)
-            .padding(.horizontal, Dimension.DetailPadding.horizontal)
+        let onSemesterListPressed: () -> Void
 
-            // MARK: - FIX
+        var body: some View {
             Button(action: {
-                path.append(StackView(type: .SemesterList))
+                onSemesterListPressed()
             }, label: {
-                Text("전체 학기 성적 보기")
-                    .font(Font.custom("Apple SD Gothic Neo", size: 15))
-                    .foregroundColor(isLatestSemesterNotYetConfirmed ? .white : Color(red: 0.15, green: 0.15, blue: 0.16))
-                    .frame(height: 39, alignment: .center)
-                    .frame(maxWidth: .infinity)
-                    .background(isLatestSemesterNotYetConfirmed ? Color(red: 0.51, green: 0.43, blue: 0.93) : Color(red: 0.95, green: 0.96, blue: 0.97))
-                    .cornerRadius(4)
+                VStack {
+                    VStack(spacing: 0) {
+                        HStack {
+                            Text("내 성적")
+                                .font(YDSFont.title3)
+                            Spacer()
+                        }
+                        .padding(.leading, 4)
+                        .padding(.top, 20)
+
+                        HStack {
+                            Image("ppussung")
+                                .resizable()
+                                .clipShape(Circle())
+                                .frame(width: 48, height: 48)
+                                .padding(.leading, 0)
+                            VStack(alignment: .leading) {
+                                Text("전체 학기")
+                                    .font(YDSFont.body2)
+                                    .padding(.bottom, -2.0)
+                                Text("성적 확인하기")
+                                    .font(YDSFont.subtitle1)
+                            }
+                            .padding(.leading, 10)
```

Comment:
```suggestion
                        HStack(spacing: 14) {
                            Image("ppussung")
                                .resizable()
                                .clipShape(Circle())
                                .frame(width: 48, height: 48)
                                .padding(.leading, 0)
                            VStack(alignment: .leading) {
                                Text("전체 학기")
                                    .font(YDSFont.body2)
                                    .padding(.bottom, -2.0)
                                Text("성적 확인하기")
                                    .font(YDSFont.subtitle1)
                            }
```

padding 대신 spacing을 활용할 수 있을 것 같아요! default 값이 4였던 것 같아서 14로 넣었습니다